Entities/MainObjects/Player.tscn

```diff
@@ -1341,11 +1341,11 @@ stream = ExtResource("58")
 bus = &"SFX"
 
 [node name="RingMagnet" type="Area2D" parent="."]
-visible = false
 collision_layer = 4096
 collision_mask = 0
 
 [node name="CollisionShape2D" type="CollisionShape2D" parent="RingMagnet"]
+visible = false
```

Comment:
explain/revert

Reply:
Opposite case with the Area2D. Reverted.